.changeset/thick-jokes-smile.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Some minor TypeScript type fixes to internal components. Also applying some ESLint recommendations.
```

Comment:
I've only applied the changeset here because in those files I changed some runtime code (e.g. `newSize + 'px'` to `${newSize}px`). Just to be safe.

In the rest only the TS types changed